docs/source/content/overview/linear_SVM.rst

```diff
@@ -2,67 +2,123 @@
+Sometimes data classes will have **outliers**. This is data points that are clearly separated from the rest of it's class. Support Vector Machines will ignore these outliers. This is shown in the figure below. 
 
-.. figure:: https://github.com/machinelearningmindset/machine-learning-for-everybody/blob/master/docs/source/content/overview/_img/SVM_Outliers.png
-   :scale: 50 %
+.. figure:: _img/SVM_Outliers.png
+   :scale: 50%
    :alt: Outliers
 
    Ref:  https://www.analyticsvidhya.com/blog/2017/09/understaing-support-vector-machine-example-code/
 
 The star that is with the red circles is the outlier. So the SVM ignores the outlier and creates the best line to separate the two classes. 
 
 
-Kernal SVM
+Kernel SVM
 -----------
 
 There will be data classes that can't be separated with a simple line or hyperplane. This is called **non-linearly separable data**. Here is an example of that kind of data. 
 
-.. figure:: https://github.com/machinelearningmindset/machine-learning-for-everybody/blob/master/docs/source/content/overview/_img/SVM_Kernal.png
-   :scale: 50 %
+.. figure:: _img/SVM_Kernal.png
+   :scale: 50%
    :alt: Outliers
 
    Ref:  https://www.analyticsvidhya.com/blog/2017/09/understaing-support-vector-machine-example-code/
 
 
-There's is no clear way to separate the stars from the circles. We will go over how SVMs will be able to classify non-lenearly separable data in our section about **Kernal SVM**. 
+There's is no clear way to separate the stars from the circles. SVMs will be able to classify non-linearly separable data by using a trick called the **kernel trick**. Basically what goes behind the scenes, is that it takes the points to a higher dimension, which turns non-linearly separable date to linear separable data. So the above figure would be classified with a circle that separates the data. 
+
+There are three types of kernels:
+
+- **Linear** Kernel
+- **Polynomial** Kernel
+- **Radial Basis Function (RBF)** kernel
+
+You can see how these kernels change the outcome of the optimal hyperplane by changing the value of kernel in "model = svm.SVC(kernel = 'linear', C = 10000)" to either 'poly' or 'rbf'. This is in the linear_svm.py. 
 
 
 Conclusion
 -----------
 
-A SVM is a great machine learning technique to classify data. The pros to SVM's are it is a great classifier when there are mulitple dimensions in the data points. It is also saves space on memory because it only uses the support vectors to create the optimal line. The cons to SVM's are that the training times are higher when there is a larger data set, so it doesn't perform well. Also it performs badly when the classes are overlapping. 
+A SVM is a great machine learning technique to classify data. Now that we know a little about SVM's we can show the advantages and disadvantages to using this classifier. 
+The pros to SVM's:
+- Effective in classifying higher dimensional space
```

Comment:
These lists do not appear correctly on the page because you need a whitespace between them and the previous paragraph